ui/raidboss/data/05-shb/trial/varis-ex.txt

```diff
@@ -0,0 +1,116 @@
+# Memoria Misera Extreme
+# -ic "Phantom Varis" -ii 4CEF 4CF1 4CDD 4CEE
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# Phase 1
+0 "Start!"
+0.0 "--sync--" sync /:Engage!/ window 0,1
+0.9 "--sync--" sync /:Varis Yae Galvus:4CEF:/ window 1,0
+19.0 "Altius" sync /:Varis Yae Galvus:4CCA:/
+28.9 "Alea Iacta Est" sync /:Varis Yae Galvus:4CD2:/
+34.1 "Terminus Est" # sync /:Terminus Est:4CB4:/
+45.1 "Citius" sync /:Varis Yae Galvus:4CF0:/
+56.0 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+64.2 "Ignis Est Action"
+64.2 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
+82.4 "Electrified Gunshield" sync /:Varis Yae Galvus:4CD7:/
+91.5 "Altius" sync /:Varis Yae Galvus:4CCA:/
+100.4 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+101.6 "Terminus Est" # sync /:Terminus Est:4CB4:/
+108.5 "Ignis Est Action" # sync /:Ignis Est:4CB6:/
+109.6 "Magitek Shock" sync /:Varis Yae Galvus:4CDA:/
+120.7 "Reinforced Gunshield" sync /:Varis Yae Galvus:4CD9:/
+128.9 "Citius" sync /:Varis Yae Galvus:4CF0:/
+132.0 "Magitek Shielding" sync /:Varis Yae Galvus:4CDB:/
+147.2 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/
+156.0 "Ventus Est Ready" sync /:Varis Yae Galvus:4CC6:/
+163.6 "Magitek Burst" sync /:Varis Yae Galvus:4CDE:/
+164.2 "Ventus Est Action"
+166.7 "Magitek Burst Hit"
+177.1 "Vivere Militare Est" sync /:Varis Yae Galvus:4CCC:/
+
+# Phase 2
+200.1 "--targetable--"
+233.2 "Vivere Militare Est" sync /:Bladesblood:4CCF:/ duration 30 window 80,50
+
+# Phase 3 (random groups)
+262.3 "Gunshield Actions" # fake action here for placeholding
+
+
+# ================= random groups ===================
+# group 1 (Loaded Gunshield, spread)
```

Comment:
I'm so sorry, I should have said more about this. What you have here will work, but there will be uncomfortable jumps at the end of each group. As you saw in the Ala Mhigo timeline, you should probably add extra lines after each group so that the user will always know what might be next. So the end of the first group might look like this:

```
317.0 "Ignis Est Action"
324.3 "Terminus Est" # sync /:Terminus Est:4CB4:/
324.5 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
330.0 "Reinforced Gunshield?"
330.0 "Electrified Gunshield?"
335.1 "Reinforcements?"
339.1 "Altius?"
341.0 "Terminus Est?"
344.2 "Aetherochemical Grenado?"
348.0 "Ventus Est Ready?"
349.2 "Terminus Est?"
```

As you can see, this weaves in the opening elements of each of the other groups. (We do not have to go farther than 6 extra lines, because by the time the Gunshield lines drop off, we will already know which block to jump to.)

Do note that the timing on the Gunshields there is made-up! I do not know exactly how long it takes from Festina Lente until the next Gunshield. You will have to determine that and ensure it matches.

Reply:
sorry, my friend and I tested more times, we found that Citius depended on which gunshield skills used, I would add Citius back and ensure the timing.

Reply:
No, no need to worry about your English abilities! It is not your fault that I didn't explain it clearly.

The groups are in a random order, yes. So at the end of each group, you would put the first 2-3 abilities from each of the other groups. In the example I gave you, Reinforcements, Terminus Est, and Aetherochemical Grenado are all from group 3, while the others are group 2. Then, at the end of group 2, you would put the first 2-3 abilities from groups 1 and 3, and the same thing for group 3.